components/tag/Tag.js

```diff
@@ -15,7 +15,7 @@ export default function Tag({ name, total, selected, onClick }) {
           selected
             ? "hover:border-white bg-tertiary-medium text-black"
             : "hover:border-tertiary-medium",
-          "flex flex-row p-1 m-2 rounded-lg text-sm font-mono border-2 cursor-pointer shadow-none"
+          "hidden md:flex flex-row p-1 m-2 rounded-lg text-sm font-mono border-2 cursor-pointer shadow-none"
```

Comment:
this will always hide all tags on mobile on any page, the issue is to hide it on the profile page only

Reply:
Ok
```suggestion
          "flex flex-row p-1 m-2 rounded-lg text-sm font-mono border-2 cursor-pointer shadow-none"
```